backend_new/src/dtos/unit-types/unit-type.dto.ts

```diff
@@ -8,10 +9,12 @@ export class UnitType extends AbstractDTO {
   @IsString({ groups: [ValidationsGroupsEnum.default] })
   @IsDefined({ groups: [ValidationsGroupsEnum.default] })
   @MaxLength(256, { groups: [ValidationsGroupsEnum.default] })
+  @ApiProperty()
   name: string;
```

Comment:
I forget if you can specify these as an enum, but if so, please use an enum (see comment on unit type factory).

Reply:
updated!